internal/aws/awsutil/conn.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// nolint:errcheck
+// nolint:gocritic
```

Comment:
This pr is not for resolve errcheck issue, so the gocritic annotation is still keeped. The gocritic issues will be resolved in another pr, since there are really a great mount of gocritic issues to be resolved.